lib/session.js

```diff
@@ -1,7 +1,7 @@
 import path from 'node:path';
 import fs from 'node:fs';
 
-import rimraf from 'rimraf';
+import { rimrafSync } from 'rimraf';
```

Comment:
Can we get rid of it in favor of `fs.rmSync`?

Reply:
Good idea, done.